src/components/structures/MatrixChat.js

```diff
@@ -210,6 +210,8 @@ module.exports = React.createClass({
         window.addEventListener('resize', this.handleResize);
         this.handleResize();
 
+        Lifecycle.initRtsClient(this.props.config.teamServerConfig.teamServerURL);
```

Comment:
Does this want to be done conditionally on there being a team server URL, as otherwise it looks like we'll always be creating a client but with an undefined URL.